tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/TaskExecutorImpl.java

```diff
@@ -16,13 +16,19 @@
 import com.google.common.base.Optional;
 
 import org.apache.commons.compress.archivers.ArchiveException;
+
+import org.codehaus.plexus.util.FileUtils;
+
```

Comment:
Are you sure? It's a unique domain, so it's surrounded by empty lines just like the other unique domains in the imports.